### PR TITLE
Infix to Postfix conversion

### PR DESCRIPTION
**Pull request by nutanaarohi123**
### Have you read the [Contributing Guidelines on Pull Requests](https://github.com/TesseractCoding/NeoAlgo/blob/master/CONTRIBUTING.md#pull-requests)?)
Yes

### Description

#617 
In Infix expression operator is in between every pair of operands. In Postfix expression, the operator is followed for every pair of operands. Infix expression is converted to postfix conversion using Stack. Postfix expression is evaluated using Stack in Left to Right order. It makes the code more efficient and even reduces the time complexity. 

**Time Complexity :** O(n)
**Space Complexity:** O(n)
The output of this code attached below.

Screenshot (64)

Screenshot (65)

 

### Checklist

- [x] I've read the contribution guidelines.
- [x] I've checked the issue list before deciding what to submit.
- [x] I've edited the `README.md` and link to my code.

## Related Issues or Pull Requests

(#617 )

### CODE REVIEW

**Comment by nutanaarohi123** [comment]
Kindly review this PR @SKAUL05 @Kajol-Kumari @iamrajiv  @plazzy99 @siddharth25pandey @atarax665 @routrohan

**Review by siddharth25pandey — APPROVED** [review]
@nutanaarohi123 Good to go
Able to merge
Thanks!

File: Python/ds/Infix_To_Postfix_Conversion.py [hunk 1]
```diff
@@ -0,0 +1,49 @@
...
+'''
+ Input:
+ Enter infix expression: A+C*(B^D-E)^(G+H*K)-J
+ Output:
+ Postfix expression:     ACBD^E-GHK*+^*+J-
+  '''
+'''
+ Time Complexity : O(n)
+ Space Complexity: Θ(n)
+'''
```
Comment (Kajol-Kumari) [inline Python/ds/Infix_To_Postfix_Conversion.py]: Please add one extra line here.

**Review by Kajol-Kumari — CHANGES_REQUESTED** [review]
@nutanaarohi123 I can see many changes in the readme. Can you please revert it and just keep the changes you made for this PR?

File: Python/ds/Infix_To_Postfix_Conversion.py (hunk 1 above)
Comment (nutanaarohi123) [reply Python/ds/Infix_To_Postfix_Conversion.py]: Updated README.md and added extra line. Kindly review @Kajol-Kumari  @atarax665

**Review by iamrajiv — CHANGES_REQUESTED** [review]
@nutanaarohi123 please format the code according to PEP8 rules. Some of the mistakes done are indentation is not a multiple of four (comment), missing whitespace around the operator, line too long, missing whitespace after ':' etc.

File: Python/ds/Infix_To_Postfix_Conversion.py [hunk 2]
```diff
@@ -0,0 +1,49 @@
...
+print('Output:')
+# displaying the Output
+print('Postfix expression: ',infix_to_postfix(infix_expression))
+'''
+ Input:
+ Enter infix expression: A+C*(B^D-E)^(G+H*K)-J
+ Output:
+ Postfix expression:     ACBD^E-GHK*+^*+J-
+  '''
+'''
```
Comment (atarax665) [inline Python/ds/Infix_To_Postfix_Conversion.py:51]: No need of starting another comment for time and space complexity. You can mention it just after sample INPUT/OUTPUT.

**Review by atarax665 — CHANGES_REQUESTED** [review]

File: Python/ds/Infix_To_Postfix_Conversion.py (hunk 2 above)
Comment (nutanaarohi123) [reply Python/ds/Infix_To_Postfix_Conversion.py:51]: okay

**Comment by iamrajiv** [comment]
@nutanaarohi123 I have formatted the code as per PEP8 rules. If you want to check whether your code is as per PEP8 rules you can take help of this http://pep8online.com/

**Comment by nutanaarohi123** [comment]
> @nutanaarohi123 I have formatted the code as per PEP8 rules. If you want to check whether your code is as per PEP8 rules you can take help of this http://pep8online.com/

Thank you.

File: Python/ds/Infix_To_Postfix_Conversion.py (hunk 2 above)
Comment (nutanaarohi123) [reply Python/ds/Infix_To_Postfix_Conversion.py:51]: > No need of starting another comment for time and space complexity. You can mention it just after sample INPUT/OUTPUT.

Updated file. kindly review @atarax665

**Review by iamrajiv — APPROVED** [review]
@nutanaarohi123 LGTM!

**Comment by nutanaarohi123** [comment]
> Kindly review this PR @SKAUL05 @Kajol-Kumari @iamrajiv  @siddharth25pandey @atarax665

**Review by SKAUL05 — APPROVED** [review]

**Pull request merged**